lib/db.js

```diff
@@ -909,12 +909,12 @@ module.exports = (
       )
   }
 
-  DB.prototype.createSigninCode = function (uid) {
+  DB.prototype.createSigninCode = function (uid, flowId) {
     log.trace({ op: 'DB.createSigninCode' })
 
     return random(config.signinCodeSize)
       .then(code => {
-        const data = { uid, createdAt: Date.now() }
+        const data = { uid, createdAt: Date.now(), flowId }
```

Comment:
I didn't realize node4 had shorthand property names (looking it up, I should, since I've already used computed property names from the same es2015).